tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/Microsoft.DotNet.SourceBuild.Tasks.csproj

```diff
@@ -1,16 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- We use dotnet-cli to build BuildTasks, so we need to disable BuildTools Roslyn properties. -->
+    <DisableBuildToolsRoslynVersion>true</DisableBuildToolsRoslynVersion>
```

Comment:
It would seem as though we may need to split dir.props up a little bit if we cannot easily import it in our projects.

Reply:
We do the same thing in `core-setup`.

With the move to using the .NET SDK's MSBuild work that Jose is doing, this should get better with time.  Eventually this can just go away.